guides/source/asset_pipeline.md

```diff
@@ -586,6 +586,19 @@ in your application are included in the `config.assets.precompile` list.
 If `config.assets.digest` is also true, the asset pipeline will require that
 all requests for assets include digests.
 
+### Raise an Error When an Asset is Not Found
+
+If you are using a recent version of sprockets-rails you can configure what happens
```

Comment:
may be specify exactly which version of sprockets-rails, like greater than > 3.something ?